docs/Developers/Newcomers_Guide.rst

```diff
@@ -109,7 +109,7 @@ strongly recommend following a tutorial, such as `this one <https://try.github.i
 
 We also have a page dedicated to git commands that will help you learn the
 basics,
-:doc:`here <Git_Basics>`.
+:doc:`on this page <Git_Basics>`.
```

Comment:
This seems to use "page" twice which seems a little grammatically strange to me. Why not make it an anonymous link?